Small adjustments across the entire document (#81)
Capstone-Skynet/Capstone-Skynet.github.io: docs/design/3_1_1_video_capture.tex

The MIPI protocol has been widely adopted for image and video transmission, and as such the camera can easily be replaced by the client in the future, if desired (fulfilling constraint \textbf{C.EX.5}).

% {Camera libraries or getting the video to a 'viewable' state}

> The \texttt{raspicam} C++ library is utilized to manage video acquisition from the camera module. \texttt{raspicam} was selected over the native Raspberry Pi camera libraries in order to perform capture and processing in a low-level C++ environment. This allows for the low-level memory manipulation/image format conversions required throughout the flow to be performed directly on the (more powerful) PMB, rather than the PLB.

Reviewer: ```suggestion
The \texttt{raspicam} C++ library manages video acquisition from the camera module. The program uses \texttt{raspicam} instead of the native Raspberry Pi camera libraries in order to perform capture and processing in a low-level C++ environment. This allows for the low-level memory manipulation/image format conversions required throughout the flow to be performed directly on the (more powerful) PMB, rather than the PLB.
```
